tests/components/media_player/test_sync_helper.py

```diff
@@ -0,0 +1,135 @@
+"""The tests for the Sync Media player helper functions."""
+
+import pytest
+
+import homeassistant.components.media_player as mp
+from homeassistant.const import (
+    STATE_IDLE,
+    STATE_OFF,
+    STATE_ON,
+    STATE_PAUSED,
+    STATE_PLAYING,
+)
+
+
+class SyncMediaPlayer(mp.MediaPlayerEntity):
+    """Sync media player test class."""
+
+    def __init__(self, hass):
+        """Initialize the test media player."""
+        self.hass = hass
+        self._volume = 0
+        self._state = STATE_OFF
+
+    @property
+    def state(self):
+        """State of the player."""
+        return self._state
+
+    @property
+    def volume_level(self):
+        """Volume level of the media player (0..1)."""
+        return self._volume
+
+    @property
+    def supported_features(self):
+        """Flag media player features that are supported."""
+        return (
+            mp.const.SUPPORT_VOLUME_SET
+            | mp.const.SUPPORT_VOLUME_STEP
+            | mp.const.SUPPORT_PLAY
+            | mp.const.SUPPORT_PAUSE
+            | mp.const.SUPPORT_TURN_OFF
+            | mp.const.SUPPORT_TURN_ON
+        )
+
+    def set_volume_level(self, volume):
+        """Set volume level, range 0..1."""
+        self._volume = volume
+
+    def volume_up(self):
+        """Turn volume up for media player."""
+        if self.volume_level < 1:
+            self.set_volume_level(min(1, self.volume_level + 0.2))
```

Comment:
Is there a reason you're choosing to up by 0.2 here, instead of the 0.1 being used in the async implementation in the main code?

Reply:
Hi, I these are old test and I just have rewritten them to the pytest style. There is probably no reason for choosing 0.2 here and not 0.1. However, I can change this to remove complexity and to make it clearer.

Reply:
Thanks. No need to do that now, but might be nice for a subsequent update.